src/app/user/Communities/CommunitiesSection.tsx

```diff
@@ -57,39 +58,39 @@ const UserCommunities = ({ nftAddresses }: Props) => {
   )
 
   if (!isConnected || (!isLoadingNfts && nftsOwned === 0)) {
-    return <JoinACommunity />
+    return <HeroCommunitiesComponent shouldShowLearnMore />
   }
 
+  const defaultCommunityVariant = nftsOwned <= 2 ? 'landscape' : 'portrait'
+  const communityGridClass = nftsOwned <= 2 ? 'xl:grid-cols-2' : 'xl:grid-cols-4'
+
   return (
     <>
-      <SectionHeader
-        name="Communities"
-        description={
-          'When you own or earn badges as part of contributions and participation in Collective Communities they will be summarized below.'
-        }
-      />
-      <div className="flex flex-wrap gap-[24px]">
-        {nftsInfo.map((nftInfo, index) => (
-          <NftInfo
-            key={nftInfo.address}
-            nftAddress={nftInfo.address}
-            onFinishedLoading={onNftFinishedLoading(index)}
-          />
-        ))}
-      </div>
+      <SectionContainer title="YOUR COMMUNITIES" headerVariant="h1">
```

Comment:
This variant should be `h3` according to designs. Continuing the previous conversation, our typography aligns with design conventions, so it's easier for devs to refer to the the variant name rather than checking its actual size.

<img width="655" height="474" alt="image" src="https://github.com/user-attachments/assets/115bdfa5-1a7f-4fa8-8139-aef59f9fb298" />

Reply:
I've adjusted it. Sometimes this is not provided in the design. I'll be sure to point these out when I find them.